wp-parsely.php

```diff
@@ -68,6 +68,10 @@ function parsely_initialize_plugin(): void {
 	$GLOBALS['parsely'] = new Parsely();
 	$GLOBALS['parsely']->run();
 
+	// Plugin dashboard page.
+	$GLOBALS['parsely_dashboard_page'] = new Dashboard_Page( $GLOBALS['parsely'] );
+	$GLOBALS['parsely_dashboard_page']->run();
```

Comment:
Can you briefly explain why we moved this here?

Reply:
I don't remember why I ended up moving this here. I think I moved it there to try to fix some issue and ended up not moving it back to the original location.

I tried moving it back to where it was and it's working, so let's keep it there :)